sessions/functionals.qmd

```diff
@@ -267,31 +269,29 @@ user_info_files
 ```
 
 ```{r admin-list-files-for-book}
-#| echo: false
 head(gsub(".*\\/data-raw", "data-raw", user_info_files), 3)
+print("...")
```

Comment:
Hmm, I'm not sure I understand the reasoning for this change, but we don't want to include the code itself, but do want to include the output (we rarely need to use `print()` in R, unlike Python). I will revert this change ☺️

Reply:
Woops, that was also not something I wanted to commit - I was testing something! Great!